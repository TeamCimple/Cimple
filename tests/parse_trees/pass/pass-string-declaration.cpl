int main() {
	string a = "Hello World";
	printf("%s\n", a);
	return 0;
}
