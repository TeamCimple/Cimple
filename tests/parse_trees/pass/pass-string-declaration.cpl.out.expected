FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations: (string EQUALS(Var(a) String("Hello World"),),  StatementList: Statement(Call(Receiver()FunctionName: printf Params: Identifier(a)String("%s\n"))), RETURN(Int(0)))) 
